app/code/pagetype/HomePage.php

```diff
@@ -5,10 +5,12 @@
 use Page;
 use SilverStripe\Assets\Image;
 use SilverStripe\Forms\TextField;
+use Sheadawson\Linkable\Models\Link;
 use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\TextareaField;
 use SilverStripe\Forms\TreeDropdownField;
 use SilverStripe\AssetAdmin\Forms\UploadField;
-use SilverStripe\Forms\TextareaField;
+use Sheadawson\Linkable\Forms\LinkField;
```

Comment:
Is there a reason to use this over site tree?

Reply:
With link, you can specify what type you want it to be and the title, without having to separate it into 2 different fields